<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncludesWidget</class>
 <widget class="QWidget" name="IncludesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>298</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KUrlRequester" name="includePathRequester">
       <property name="toolTip">
        <string comment="@info:tooltip">Add the currently displayed URL to the list of includes if it is not in the list yet.</string>
       </property>
       <property name="placeholderText">
        <string comment="@info:placeholder">A new include path/file to use for parsing files under given directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addIncludePath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string comment="@info:tooltip">Create a new include path entry from this URL</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-add"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeIncludePath">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string comment="@info:tooltip">Delete current include path entry</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-remove"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="includePaths">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>8</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KMessageWidget" name="errorWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>KUrlRequester</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>KMessageWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
